Resources/Prototypes/Recipes/Construction/utilities.yml

```
@@ -874,7 +874,7 @@
 
 # FAN
 - type: construction
-  name: Little fan
+  name: Little directional fan
```

```suggestion
  name: little directional fan
```
Тут была ошибка локализации, рецепты все с маленькой буквы пишутся